src/adapters/Database.re

```diff
@@ -83,33 +85,34 @@ let addHelpItem = (userId, description, sendMessageWithAttachments) => {
   );
 };
 
-let addRoom = (itemId, room, sendMessage) => {
-  let conn = connect();
+let addRoom = (itemId, room) =>
+  Js.Promise.make((~resolve as _resolve, ~reject as _reject) => {
```

Comment:
the reason `reject` typed as `_reject` is to suppress warnings from the compiler (no unused variables), so it's not needed on `resolve` as `resolve` is actually used!

Reply:
Yup! Functions in Reason are curried by default as well, I believe that's why we still need to grab  `~reject` (`~` denotes a named argument so you don't have to worry about order) here! I'll have a look tomorrow to make sure!